errors.json

```diff
@@ -1,19 +1,19 @@
   "6": "You may not unsubscribe from a store listener while the reducer is executing. See https://redux.js.org/api/store#subscribelistener for more details.",
-  "7": "Actions must be plain objects. Use custom middleware for async actions.",
-  "8": "Actions may not have an undefined \"type\" property. Have you misspelled a constant?",
+  "7": "Actions must be plain objects. Instead, the actual type was: ''. You may need to add middleware to your store setup to handle dispatching other values, such as 'redux-thunk' to handle dispatching functions. See https://redux.js.org/tutorials/fundamentals/part-4-store#middleware and https://redux.js.org/tutorials/fundamentals/part-6-async-logic#using-the-redux-thunk-middleware for examples.",
+  "8": "Actions may not have an undefined \"type\" property. You may have misspelled an action type string constant.",
   "9": "Reducers may not dispatch actions.",
-  "10": "Expected the nextReducer to be a function.",
-  "11": "Expected the observer to be an object.",
-  "12": "bindActionCreators expected an object or a function, instead received . Did you write \"import ActionCreators from\" instead of \"import * as ActionCreators from\"?",
-  "13": "Dispatching while constructing your middleware is not allowed. Other middleware would not be applied to this dispatch.",
-  "14": "Reducer \"\" returned undefined during initialization. If the state passed to the reducer is undefined, you must explicitly return the initial state. The initial state may not be undefined. If you don't want to set a value for this reducer, you can use null instead of undefined.",
-  "15": "Reducer \"\" returned undefined when probed with a random type. Don't try to handle  or other actions in \"redux/*\" namespace. They are considered private. Instead, you must return the current state for any unknown actions, unless it is undefined, in which case you must return the initial state, regardless of the action type. The initial state may not be undefined, but can be null.",
-  "16": "Super expression must either be null or a function"
+  "10": "Expected the nextReducer to be a function. Instead, received: '",
+  "11": "Expected the observer to be an object. Instead, received: ''",
+  "12": "The slice reducer for key \"\" returned undefined during initialization. If the state passed to the reducer is undefined, you must explicitly return the initial state. The initial state may not be undefined. If you don't want to set a value for this reducer, you can use null instead of undefined.",
+  "13": "The slice reducer for key \"\" returned undefined when probed with a random type. Don't try to handle  or other actions in \"redux/*\" namespace. They are considered private. Instead, you must return the current state for any unknown actions, unless it is undefined, in which case you must return the initial state, regardless of the action type. The initial state may not be undefined, but can be null.",
```

Comment:
> Don't try to handle  or other actions in \"redux/*\" namespace.

Is this missing a char or '' to get replaced that I'm not seeing?

Reply:
There's a template literal interpolation there:

```js
`Don't try to handle ${ActionTypes.INIT} or other actions
```

and that doesn't get included in the extracted message.

I'll add single quotes around it, though.